testapi.pm

```diff
@@ -1904,10 +1856,7 @@ Trigger backend specific power action, can be C<'on'>, C<'off'>, C<'acpi'> or C<
 
 =cut
 
-sub power {
-
-    # params: (on), off, acpi, reset
-    my ($action) = @_;
+sub power ($action) {
```

Comment:
```
tests/xfstests/run.pm
474:            power('reset', keepconsole => is_pvm);
```

Reply:
yeah I think someone introduced dead code that is never called. Because this should likely be

```
power_action('reset', keepconsole => is_pvm);
```

Reply:
Adressed in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/14996 but as explained not blocking as it's obviously dead-code